scapy/contrib/automotive/obd/obd.py

```diff
@@ -62,13 +62,14 @@ def answers(self, other):
 
 
 # Service Bindings
+
 bind_layers(OBD, OBD_S01, service=0x01)
 bind_layers(OBD, OBD_S02, service=0x02)
 bind_layers(OBD, OBD_S03, service=0x03)
 bind_layers(OBD, OBD_S04, service=0x04)
 bind_layers(OBD, OBD_S06, service=0x06)
 bind_layers(OBD, OBD_S07, service=0x07)
-bind_layers(OBD, OBD_S08, service=0x08)
+bind_layers(OBD, OBD_TID_Record, service=0x08)
```

Comment:
Isn't it weird that each class name contains its service number but `OBD_TID_Record `does not?
Is there are reason for this?
The same for `OBD_MID `and `OBD_TID `

Reply:
I agree. I changed it.